apps/src/applab/applab.js

```diff
@@ -814,13 +815,26 @@ function setupReduxSubscribers(store) {
   });
 
   if (store.getState().pageConstants.hasDataMode) {
+    if (experiments.isEnabled(experiments.APPLAB_DATASETS)) {
+      const sharedTablesRef = getSharedDatabase().child('counters/tables');
```

Comment:
Can you factor this out into a helper function? This section and the one below look pretty similar.